lib/multi-level.js

```diff
@@ -36,28 +57,26 @@ class MultiLevelCache {
     this.shouldCache = opts.shouldCache || (() => true);
 
     this._pendingRefreshes = {};
-    this._initTask = Promise.all(opts.caches.map(cache => {
-      return cache.init().catch(err => {
-        throw err;
-      });
-    }));
+    this._initTask = Promise.all(opts.caches.map(cache => cache.init()));
   }
 
   /**
    * Attempts a cache get
    *
-   * @param {String}   key                    - The cache key
-   * @param {Object}   opts                   - Options for this particular read
-   * @param {Boolean}  [opts.skipCache=false] - Whether the cache should be bypassed (default false)
-   * @param {String}   [opts.maxAge]          - The duration in milliseconds before a cached item expires
-   * @param {ShouldCache} [opts.shouldCache]  - A function to determine whether or not we should cache the item
-   * @param {UpdateFn} updateFn               - async function that defines how to get a new value
+   * @param {String} key The cache key
+   * @param {Object} [opts={}] Options for this particular read
+   * @param {Boolean} [opts.skipCache=false] Whether the cache should be bypassed (default false)
+   * @param {String} [opts.maxAge] The duration in milliseconds before a cached item expires
+   * @param {Number} [opts.maxStaleness=0] The duration, in milliseconds, in which expired cache items are still served
+   * @param {ShouldCache} [opts.shouldCache] A function to determine whether or not we should cache the item
+   * @param {UpdateFn} updateFn async function that defines how to get a new value
    *
    * @async
    * @returns {Promise<GetResult>} a Promise which resolves to an object containing
    * a `value` property and a `fromCache` boolean indicator.
    */
   async get(key, opts, updateFn) {
```

Comment:
Maybe something like [optional-options](https://www.npmjs.com/package/optional-options) could be used to avoid having to pass in an empty object. 😉

Reply:
it's not a callback though, and while it has the same shape as what optional-options wants, its semantically different.  Really the right thing to do is making a breaking change and reverse the order of the params.  We should consider doing that at some point. Not sure its warranted just yet though. something to bear in mind (I'll open an issue)

Reply:
[Issue](https://github.com/godaddy/out-of-band-cache/issues/6)